src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,107 @@
-export const Pagination = () => {};
+import { getNumbers } from '../../utils';
+
+interface Props {
+  items: string[];
+  itemsPerPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  items,
+  itemsPerPage: ItemsPerPage,
```

Comment:
The destructured `itemsPerPage` prop is renamed to `ItemsPerPage`, which is not a conventional practice and may lead to confusion. It's better to keep the original name `itemsPerPage` for consistency.